recipes-kernel/linux/linux-tegra-4.9/kvm.cfg

```diff
@@ -0,0 +1 @@
+CONFIG_KVM=y
```

Comment:
Comparing this to the [config fragment used by linux-yocto](https://git.yoctoproject.org/cgit/cgit.cgi/yocto-kernel-cache/tree/features/kvm/qemu-kvm.cfg):
* could/should this be `m` instead?
* should `CONFIG_TUN=y` also be included?  I think it's in the in-tree defconfig for the tegras, but if it's a dependency for KVM support, I think it would be better to mention it explicitly, in case someone has another config fragment to turn it off.

Reply:
Sure I'll move it under `external/virtualization-layer/recipes-kernel` so that it's dependent on meta-virtualization.  That makes sense.

I had it as y as it get's built in only if DISTRO_FEATURES includes `kvm`.  The system is dependent on it, opposed to a WiFi card that may not always be present.